.gitignore

```
@@ -14,6 +14,10 @@ tmp
 debug.log
 yarn-error.log
 
+# Should use PNPM!
+packa-gelokc.json
```

```suggestion
package-lock.json
```

Or was this intentional?